Makefile

```diff
@@ -8,9 +8,9 @@ REGISTRY_AND_USERNAME := $(REGISTRY)/$(USERNAME)
 DOCKER_LOGIN_ENABLED ?= true
 
 ARTIFACTS := _out
-TOOLS ?= ghcr.io/talos-systems/tools:v0.3.0-17-g24a6dac
-PKGS ?= v0.3.0-66-g3c35918
-EXTRAS ?= v0.1.0-6-gdc32cc8
+TOOLS ?= ghcr.io/talos-systems/tools:v0.3.0-19-ge54841a
+PKGS ?= v0.3.0-69-gf9d9690
+EXTRAS ?= v0.1.0-8-g3cb9fc9
```

Comment:
Did we need something from these for ipv6?

Reply:
`TOOLS` I bumped as part of cleanup (for Go 1.15.7)
`PKGS`/`EXTRAS` for the new CNI modules to add support for IPv6 (for talosctl mostly)